content/en/contribute/development.md

```diff
@@ -233,19 +246,40 @@ We have developed a [separate Hugo documentation contribution guide][docscontrib
 While making changes in the codebase it's a good idea to build the binary to test them:
 
 ```
-go build -o hugo main.go
+mage hugo
+```
+
+This command generates the binary file at the root of the repository.
+
+If you want to install the binary in `$GOPATH/bin`, run
+
+```
+mage install
 ```
 
 ### Test 
 Sometimes changes on the codebase can cause unintended side effects. Or they don't work as expected. Most functions have their own test cases. You can find them in files ending with `_test.go`.
 
-Make sure the commands `go test ./...` passes, and `go build` completes.
+Make sure the commands 
+
+```
+mage hugoRace
+mage -v check
```

Comment:
I think this is a little "too much". `mage -v check` should be enough.

Reply:
I see. I deleted `mage -v check`.